rotkehlchen/chain/evm/tokens.py

```diff
@@ -97,10 +97,10 @@ def generate_multicall_chunks(
     return multicall_chunks
 
 
-class EthTokens():
-    def __init__(self, database: DBHandler, ethereum: EthereumManager):
+class EvmTokens():
+    def __init__(self, database: DBHandler, manager: EthereumManager):
```

Comment:
I guess it would make sense to also rename the EthereuManager class. I've been checked and all the methods should be available in other chains with the exception of the ENS lookup. The method that makes usage of the subgraph should be removed soon too.

Reply:
Yeah I'm making small PRs for easier review. That one will come soon :smiley: